commcare_connect/opportunity/helpers.py

```diff
@@ -1,42 +1,58 @@
 from django.db.models import Case, Count, F, Max, Min, Q, Sum, When
 
-from commcare_connect.opportunity.models import Opportunity, OpportunityAccess, VisitValidationStatus
+from commcare_connect.opportunity.models import Opportunity, OpportunityAccess, UserInvite, VisitValidationStatus
 
 
 def get_annotated_opportunity_access(opportunity: Opportunity):
```

Comment:
Might be good to add test-coverage for this if not present already

Reply:
We have tests for this function in the `commcare_connect/opportunity/tests/test_export.py` file.